modules/materials-page/materials-page.service.js

```diff
@@ -0,0 +1,50 @@
+const Materials = require('./materials-page.model');
+const createTranslations = require('../../utils/createTranslations');
+const {
+  addTranslations,
+  updateTranslations,
+  deleteTranslations,
+} = require('../translations/translations.service');
+
+class MaterialsService {
+  async getAllMaterialsBlocks() {
+    return Materials.find({}).exec();
+  }
+
+  async addMaterialsBlock(materialsBlock) {
+    materialsBlock.translationsKey = await addTranslations(
+      createTranslations(materialsBlock)
+    );
+
+    return new Materials(materialsBlock).save();
+  }
+
+  async deleteMaterialsBlock(id) {
+    const materialsBlock = await Materials.findByIdAndDelete(id);
+
+    if (materialsBlock) {
```

Comment:
I think here must be else case with throwing error or returning false in case of materialBlock doesn't exist

Reply:
fix)